pkg/resources/server.go

```diff
@@ -309,6 +322,14 @@ func (rs *resourceServer) restart() error {
 	// Send terminate signal to ListAndWatch()
 	rs.termSignal <- true
 
+	// wait for the terminated signal or 5 second
```

Comment:
In `resoruceServer.Stop()` we do the `termSignal-terminatedSignal` handshake before stopping the grpcServer. n restart, we do it after `rs.grpcServer.Stop()`. Is it intentional?

Reply:
you are right! sorry